00_hello_world/main.go

```diff
@@ -8,6 +8,19 @@ import (
 
```

Comment:
Move your code from `00_hello_world` to `01_fib`

Reply:
Done